### ISSUE TITLE
Run command in document from editor

### ISSUE THREAD

**Issue by ahus1** [asciidoctor/asciidoctor-intellij-plugin#977]
## Why the new feature should be added

In 2022.1 EAP, the Markdown plugin has the following feature: 

> We’ve implemented some improvements for working with Markdown files. 
If a Markdown file contains instructions with commands that you need to execute in your terminal, you can run those commands directly from the Markdown file using the run icons in the gutter.

![image](https://user-images.githubusercontent.com/3957921/151601527-89480896-c3fc-46df-b5c9-9b7801dcb8bd.png)

https://blog.jetbrains.com/idea/2022/01/intellij-idea-2022-1-eap-1/

## How the new feature should work

It should work like it works for the Markdown plugin :-)

**Comment by Mogztter** [comment]
Looking forward to this feature 😉

**Comment by ahus1** [comment]
There is now pre-release 0.37.14 that has some support for this. I decided to implement it only for the editor, not for the preview. If you think this would be useful for the preview as well, please let me know. 

This will work also in IntelliJ 2021.3.1 for the AsciiDoc plugin.

The following snippet: 

```
Let's run `mvn compile`!

[source,bash]
----
echo "hello"
----
```

Will display as: 

![image](https://user-images.githubusercontent.com/3957921/163690132-3b4cae96-7738-44f6-a928-8b333c595640.png)

In addition to "bash" also "jshell" works. 

[I wrote some bits in the docs when this works and how](https://intellij-asciidoc-plugin.ahus1.de/docs/users-guide/features/editor.html#interaction). 

I found that when there are two commands in a single line, none of them can be executed at the moment. I wished I tried this feature earlier. Let's hope there'll be a fix or workaround soon: https://youtrack.jetbrains.com/issue/IDEA-292343

The pre-release of the plugin is available from [GitHub releases](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/installing-a-preview-release.html) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).

Please give it a try and comment here with your findings. I'll leave this issue open for a while. Thanks!

**Comment by mojavelinux** [comment]
Wow!

**Comment by ahus1** [comment]
This is now available in the general release 0.37.17.
